hack/e2e.sh

```diff
@@ -19,7 +19,7 @@ set -o errexit   # exits immediately on any unexpected error (does not bypass tr
 set -o nounset   # will error if variables are used without first being defined
 set -o pipefail  # any non-zero exit code in a piped command causes the pipeline to fail with that code
 
-KIND_VERSION="v0.5.1"
+KIND_VERSION="v0.7.0"
```

Comment:
Was the KIND bump brought in through the CAPI bump ? if not I’d rather see this in a specific commit

Reply:
Hi @yastij,

You hit the nail on the head. The updated CAPI e2e kind bits required the newer version of Kind.